Sources/UtiliKit/General/LanguageTool.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  LanguageTool.swift
+//  UtiliKit-iOS
+//
+//  Created by Russell Mirabelli on 10/11/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
```

Comment:
Copyright should be changed
```suggestion
//  Copyright © 2018 Bottle Rocket Studios. All rights reserved.
```